packages/core/echo/echo-db/src/database.ts

```diff
@@ -179,7 +186,7 @@ export class EchoDatabaseImpl extends Resource implements EchoDatabase {
       return undefined;
     }
 
-    const object = defaultMap(this._rootProxies, core, () => initEchoReactiveObjectRootProxy(core));
+    const object = defaultMap(this._rootProxies, core, () => initEchoReactiveObjectRootProxy(core, this));
```

Comment:
extract `defaultMap(this._rootProxies, core, () => initEchoReactiveObjectRootProxy(core, this));`